saleor/product/models.py

```diff
@@ -283,6 +283,7 @@ class Product(SeoModel, ModelWithMetadata, PublishableModel):
     )
     objects = ProductsQueryset.as_manager()
     translated = TranslationProxy()
+    rating = models.FloatField(null=True, blank=True)
```

Comment:
We should move `rating` above `objects`. :police_officer:

Reply:
We should separate objects and translated from the model fields 👍